src/Resources/PageResource.php

```diff
@@ -139,18 +149,15 @@ public function getDefaultPageStatus(): string
      */
     public function getLanguageOptions(): array
     {
-        return [
-            'nl' => 'Nederlands',
-            'en' => 'English',
-        ];
+        return config('nova-pages-tool.languages');
     }
 
     /**
      * Extend this to provide your own default language.
      */
     public function getDefaultLanguage(): string
     {
-        return config('app.locale');
+        return config('nova-pages-tool.defaultLanguage');
```

Comment:
Ik zie hier de meerwaarde niet van. Kan je dat uitleggen?

Reply:
Dit is niet meer van toepassing. We gaan naar een config file.